### PR TITLE
Sidepanel: Close after tx signed

### LINKED ISSUE #211: Close panel after user signs transaction
See https://www.notion.so/9e7e5cd565b44800a79b5640a5f094a8?v=d3b5e76c06634dafa4b7083edbaca881&p=2da0ba795edc4c4290605d573ff0f31d

### PR DESCRIPTION
**Pull request by fabriziovigevani**
closes #211 

Once we have an implementation of the `Signerpanel` we can show proper messages in case the tx failed

### CODE REVIEW

File: src/components/Disputes/panels/AppealPanel.js [hunk 1]
```diff
@@ -112,8 +112,9 @@ const AppealPanel = React.memo(function AppealPanel({
         dispute.lastRoundId,
         appealOption.outcome
       )
-      await tx.wait()
+
```
Comment (rperez89) [inline src/components/Disputes/panels/AppealPanel.js]: are we adding those blank lines between for some reason?

File: src/components/Disputes/panels/RevealPanel.js [hunk 2]
```diff
@@ -39,9 +39,10 @@ const RevealPanel = React.memo(function RevealPanel({
         jurorDraft.commitment,
         password
       )
+
```
Comment (rperez89) [inline src/components/Disputes/panels/RevealPanel.js]: same question here

File: src/components/Disputes/panels/AppealPanel.js (hunk 1 above)
Comment (fabriziovigevani) [reply src/components/Disputes/panels/AppealPanel.js]: Ah yes, i added them for better readability

File: src/components/Disputes/panels/AppealPanel.js [hunk 3]
```diff
@@ -112,8 +112,9 @@ const AppealPanel = React.memo(function AppealPanel({
...
       onDone()
+      await tx.wait()
     } catch (err) {
       console.log('Error submitting tx: ', err)
```
Comment (rperez89) [inline src/components/Disputes/panels/AppealPanel.js]: should we use `console.error` for errors?

Comment (fabriziovigevani) [reply src/components/Disputes/panels/AppealPanel.js]: Yes, i think we should. Initially this was kind of a placeholder to catch the error thinking that in the future we would be handling this in the signer panel.

Comment (rperez89) [reply src/components/Disputes/panels/AppealPanel.js]: cool, so in the meanwhile we can change those `console.log` for `console.error` right?

**Pull request merged**